packages/vite/src/node/ssr/inlineImport.ts

```diff
@@ -0,0 +1,76 @@
+import type { InlineConfig } from '../config'
+import { resolveConfig } from '../config'
+import { createRunnableDevEnvironment } from '../server/environments/runnableEnvironment'
+import { mergeConfig } from '../utils'
+
+interface InlineImportResult<T> {
+  module: T
+  dependencies: string[]
+}
+
+/**
+ * Import any file using the default Vite environment.
+ * @experimental
+ */
+export async function inlineImport<T>(
```

Comment:
Is there a reason this is named `inlineImport`? (What does the "inline" refer to) Since this is used for `configLoader=runner`, does it make sense to name this as `runnerImport` instead?

Reply:
done